controllers/tools/views/user-dashboard.njk

```diff
@@ -0,0 +1,202 @@
+            <a href="./dashboard" class="btn btn-dark mx-sm-3">Clear search</a>
+        </div>
+    </form>
+    <small id="emailHelp" class="form-text text-muted">You can search by a whole email address or just part of one, eg. "david" or "@gov.uk".</small>
+{% endmacro %}
+
+{% macro pendingAppsList(pendingApps) %}
+    <h2 class="t3 u-margin-top-l">
+        Funding applications by {{ singleUser.username }}
+    </h2>
+
+    <h3 class="t4 u-margin-top">
+        Pending applications
+        <small>{{ pendingApps.length }} total</small>
+    </h3>
+    {% if pendingApps.length > 0 %}
+        <table class="table table-striped table-bordered table-hover">
+            <thead class="thead-light">
+            <tr>
+                <th scope="col">Project name</th>
+                <th scope="col">Application type</th>
+                <th scope="col">Status</th>
+                <th scope="col">Create / update / expiry dates</th>
+            </tr>
+            </thead>
+            <tbody>
+            {% for app in pendingApps %}
+                <tr class="{{ statusClass(app.currentProgressState) }}">
+                    <td>{{ app.applicationData.projectName or 'Untitled' }}</td>
+                    <td>{{ deSlugify(app.formId) }}</td>
+                    <td>
+                        {{ deSlugify(app.currentProgressState) }}
+                    </td>
+                    <td>
+                        <strong>Created on</strong>: {{ dateItem(app.createdAt) }}<br />
+                        <strong>Updated on</strong>: {{ dateItem(app.updatedAt) }}<br />
+                        <strong>Expires on</strong>: {{ dateItem(app.expiresAt) }} ({{ app.expiresAtRelative }})
+                    </td>
+                </tr>
+            {% endfor %}
+            </tbody>
+        </table>
+    {% else %}
+        <p>No pending applications found for this user.</p>
+    {% endif %}
+{% endmacro %}
+
+
+{% macro userSearchResults(users) %}
+    <h2 class="t3 u-margin-top-l">
+        Search results for "{{ usernameSearch }}"
+        <small>{{ users.length }} users found</small>
+    </h2>
+
+    {% if users.length > 0 %}
+        <table class="table table-striped table-bordered table-hover">
+            <thead class="thead-light">
+            <tr>
+                <th scope="col">Email address</th>
+                <th scope="col">Activated?</th>
+                <th scope="col">In password reset mode?</th>
+                <th scope="col">Actions</th>
+            </tr>
+            </thead>
+            <tbody>
+            {% for user in users %}
+                <tr>
+                    <td>
+                        {{ user.username }}<br />
+                        <small>
+                            <strong>User ID</strong>: {{ user.id }} <br />
+                            <strong>Registered</strong>: {{ dateItem(user.createdAt) }} <br />
+                            <strong>Last login</strong>: {{ dateItem(user.updatedAt) }}
```

Comment:
Don't think `updatedAt` is the same as last login. Most likely when the record was last updated via  password reset, change of email etc.